### ISSUE TITLE
pulp module missing solvers

### ISSUE THREAD

**Issue by finsnoop** [coin-or/pulp#286]
Tried installing scikit-criteria on Debian 10 using `pip install scikit-criteria` and everything works just perfect until I try to import SIMUS class using `from skcriteria.madm.simus import SIMUS`

I get an attribution error: `AttributeError: module 'pulp' has no attribute 'solvers'
`
I am running Python 3.8 and I have tried

- [ ] Tried out the latest github version: https://github.com/coin-or/pulp
- [ ] Searched for an existing similar issue: https://github.com/coin-or/pulp/issues?utf8=%E2%9C%93&q=is%3Aissue%20

**Comment by pchtsp** [comment]
I just checked their code.
In https://github.com/leliel12/scikit-criteria/blob/fed3262e1ed00cf8eda0164d3e918aff4a654023/skcriteria/utils/lp.py#L58

1. They use a correct but not standard reference (`pulp.solvers.SOME_SOLVER`) when it should have been `pulp.SOME_SOLVER`.
2. We have now changed the files format, so the absolute reference should be `pulp.apis.SOME_SOLVE`. Nevertheless, `pulp.SOME_SOLVER` still works and should be encouraged.

As a fix, you could try opening an issue in their github so they take out the "solvers" part and leave it with `pulp.SOME_SOLVER`.

@stumitchell maybe we could create a file named `solvers` with a deprecated reference to `apis` in the current version in case there are other projects that do this so they do not fail?

**Comment by stumitchell** [comment]
@pchtsp I think we are fine, 

I have reported it to the other library maintainer
https://github.com/leliel12/scikit-criteria/issues/12

In the mean time @finsnoop please try installing an older version of pulp 1.6.10 should do

**Comment by finsnoop** [comment]
Thank @pchtsp  and @stumitchell I have written to them about this.

**Comment by pchtsp** [comment]
ok, we'll close it here. Feel free to re-open it if you need anything.

**Comment by iamz33** [comment]
Is this issue solved yet? I'm facing same issue. Switching to older version of pulp is raising other errors. 

> ImportError: cannot import name 'clock' from 'time' (unknown location)

I'm not a programmer so please help in layman's terms.

**Comment by rhardman90** [comment]
I have this error today too - e.g. 
running `pulp.listSolvers()` is raising the error `AttributeError: module 'pulp' has no attribute 'listSolvers'`
